dataquality/clients/objectstore.py

```diff
@@ -11,26 +12,31 @@
 from dataquality.core.auth import api_client
 from dataquality.utils.file import get_file_extension
 
+IMAGES_BUCKET_NAME = "galileo-images"
 ROOT_BUCKET_NAME = "galileo-project-runs"
 RESULTS_BUCKET_NAME = "galileo-project-runs-results"
 
 
 class ObjectStore:
+    IMAGES_BUCKET_NAME = IMAGES_BUCKET_NAME
     ROOT_BUCKET_NAME = ROOT_BUCKET_NAME
     RESULTS_BUCKET_NAME = RESULTS_BUCKET_NAME
     DOWNLOAD_CHUNK_SIZE_MB = 256
 
-    def create_project_run_object(
+    def create_object(
         self,
         object_name: str,
         file_path: str,
         content_type: str = "application/octet-stream",
         progress: bool = True,
+        bucket_name: Optional[str] = None,
     ) -> None:
+        if not bucket_name:
+            bucket_name = self.ROOT_BUCKET_NAME
         url = api_client.get_presigned_url(
             project_id=object_name.split("/")[0],
             method="put",
-            bucket_name=self.ROOT_BUCKET_NAME,
+            bucket_name=bucket_name,
```

Comment:
could be fancy like and remove the top `if` 😎 
```suggestion
            bucket_name=bucket_name or self.ROOT_BUCKET_NAME,
```

Reply:
pinkies up!